scripts/rabbitmq-server.bat

```diff
@@ -48,9 +48,97 @@ if not exist "!ERLANG_HOME!\bin\erl.exe" (
 
 set RABBITMQ_EBIN_ROOT=!RABBITMQ_HOME!\ebin
 
+if not exist "!RABBITMQ_SCHEMA_DIR!" (

```

Comment:
What do you think about combining the configuration setup code into its own `.bat` file to be shared between `rabbitmq-server.bat` and `rabbitmq-service.bat`?

Reply:
We do have `rabbitmq-env.bat`. Do you think it can be moved there?

Reply:
Let's wait for future powershell work